bot/client.js

```diff
@@ -6,10 +6,17 @@ const { isAdmin } = require('./botUtils.js');
 if (process.env.NODE_ENV === 'production' && !process.env.DEBUG_MODE) process.env.DEBUG_MODE = 0;
 
 const client = new Discord.Client();
-
+const channelId = '442034467482370052';
 // Pre-load controllers
 const controllers = require('./controllers')();
 
+// Sending the message to the user when they add
```

Comment:
Remove the old Welcome Feature, this should only include code for the verify in dm feature.